test/date_range_test.rb

```diff
@@ -0,0 +1,137 @@
+require_relative 'test_helper'
+
+
+describe 'Date class' do
+  let (:date_range) {
+    Hotel::DateRange.new((Date.today), (Date.today + 3))
```

Comment:
Unnecessary parenthesis (it's already unambiguous):

```suggestion
    Hotel::DateRange.new(Date.today, Date.today + 3)
```